manifests/m/Microsoft/Sysinternals/BGInfo/4.32/Microsoft.Sysinternals.BGInfo.installer.yaml

```diff
@@ -3,13 +3,17 @@
 
 PackageIdentifier: Microsoft.Sysinternals.BGInfo
 PackageVersion: 4.32
+InstallerType: portable
+Commands:
+- bginfo

```

Comment:
```suggestion
- bginfo /accepteula
```

Reply:
This is necessary to clear the first-time EULA window:
![image](https://user-images.githubusercontent.com/114952565/217399488-2e6c7c39-6012-4ca1-9163-13fce307062c.png)